ietf/static/img/iab-logo.svg

```diff
@@ -0,0 +1,9 @@
UJBIRgkJqbV0S0uDgJWVTNiaikZE3cxEaFVF3dwr27euBTZs2KRkHDxPS0aNPQ3PzlZwk2VQtFhEhOSGxMZyBO+rB9DFFa+CkY/dL3EGmbPlMIbUpkrIvt8jo44/hQE0NfK+zMx4hiUTE/rp2dpZAff089PcvCYn91e3sVCckJxE9+eQ/QZz1tMfGxuDy5T9C1Zp10NxyNSej2ERkceOTcUiZRfOZzGL/+Lj5jx7fRiMet/KwXT6mJtgWqI2TWBM5BX5SoRj8tilSMf6gtbSltly6ZZrGGvhdvw53nzkDY62t8Hnoz4TU3z8X6ZTNTkStrSmorPw2DA7W3VLrh72GvTbqKVvSRfTss89BeuO3wdzwI/i0uAUqNn03XhG9dwpm//c3NzOZxS2jo55FBNzZmj1kMeEO0rsecTurRqG5Hjx4q6zcRpufsz0BCdsmKcz4ndoa4TY6uHH4QWrLJ9uzaSx3tG7dUu6IHoQ9PaVw6tTPoLVV3qlrJjh20ItE1Ni4CwYGTttO35igRkbqIomQ8kVExTX3xDYOCorINLPb7Np320DP5tgtbgRyVg3ImiO3yzcarHVLdn3G+O35OUPm9Np3BRHH16xIgR8PjTDAOjNFE8B+xu+0bTvx0O3bnU3z+t067YsjwnVGGCGNj4sjpPvvvx8G+IROQJiIWlv/xlFEkMsXVeaEw8SjIkLSIvJHCBEBF+k4nbWiz3mNjrAtkOiAOBfhBbiy2iQFGb/dtvsESe9zfhYmckht+eS4AptFSBUV8O6GDVDPFrHxZ5COHXsBOjuDt9RaEtFWaG+fcxQRRUWEpEXkj5AiUkkcZUlktknyO367bTeR/JOsM3KhWz55uRyk8rbb4GJDA2yUKSQUUU/PHBw4sPSYm4iQKIWkReSPPBKRJuF4vVC2ctUqOLd+PTRSIbF8Unt7Cvr7D/kSEoqITflwyuVVRIhsIbHP6+n5gRaRD7SINDLxVc+IXWfErjeiQmIn15hEvApJhogQWUJa+pytuSsRtIi8oUWkkY2vVkVs8RpbxMauN6IXabIDuKenGwYGjji+X6aIQFJSW4vIP1pEmijw3TctqJBkiwgJIyQtIv9oEWmiIlATR79CikpESBAhaRH5R4tIEyWBO8p6FdLg4GCkIkL8CEmLyD9aRJqoCd1R1imp3d6+B44ffw2OHYNIRURxS2qfOnUB2tt3ahH5QJGIGhzKTahsVSSLBq7oWb5f2Bs5oWUEDkLq6SmGzs4FZSJCnIR0/DjAPfektIg8ojAi2k+uvRIxaq0gzpcr3en+8KuUNQKkyAhshERRJSLETkhsbRSbTmoRuaN4auYmI2SPxEJnUaJl5JPAOSMeUQ4JUS0icMghsWvttIjciTlHtMO6MNOwLrSk0VASWxZpJCBNRmAjpDhEhNgJCRInome1iOwZ5S7kFOWVsNIglq046lBaA8tvvEFev9/mc2kpVSBXvfNXvneRzzsaoj7QsqZY9s4zIbEpW0sL7GL1jx94AOArX7lPuYgQFFJLy13Q2Tmek1CSRPTMM88sAhQtzi+YxdL/MwKQ0LNm9ODmE9l2rXK6rOncXq5Q/LsC8bRZgtnBXeDZQaTWwLXqwTIbRwXRWlc+tZVOClIjIwQjpDvuAJiYKP7NuXPDWdV1tHmy2VXZM2eKbrJp2uOPg1leXgOrV6+ObTyWiLJlZWX/8NlnN7/+6e/PzN38JN7vKEEiohHIWVLrCLgpm6iNTy8RFl+/h9Zm7rPkg1KpcslZiVr1dJHH+e3r6Mgn0hLYIgzDaDVNc7C2tvbxTCZzaGhoKKW62whLZLe0tGQnJibevHr16g82b4aD770Hr6bT6VcrKyvr9u3bl2shrRIiou4bN27kFmSxHnVGUcnFiq/uLl25Rn1HlgSIyEsCm28KaNfGh7Y8AtKn364jKv0cWkTMrVUPLWK2h+tiQrevE9geiFRGlDiEREU0Pj5+S1F4VokgnU5fUi0kkYjImGIRUkIiIiojfl0Ozf/0kRK0+Msr6qdG+4qJCoA1kZKsbTYVDamMRBULnbZvJ0qNDZFM00Rcu3btSDqd7m5ublYyZXMSEVjVLDOZzN3T09Njzz33HMzMzEQ+JicRWWMaMRfnt6qcsiU0R9TL1b+uJiIQJZtF+Rm+mBj+e9YqX4vJ5oMe62A7LboUbV/njHyiTEagUEhuIkJUCslNRGRMyoSUR5d48JUDPTcGJO+vspl29UpYSCmqT21Xs1pjg1IZgQIheRURokJIXkVExhS5kPLwWjORgDD6aRMc/DSBPExKrYI11dtL+tb7lRtPk2D7UXcdKTiUywgiFJJfESFRCsmviMiYIhNSHoioieRxcA0PymWK5F9EbXzAeg9tCz3sIJz9IcTBbx+FtF+fTfOPsgS2CJlJ7aAioshOagcVETcmqUntBIvI6+Uge8m0ij9rJQKTxw3Wa5FRroA8foZdAlvUqqeNW3ZAmdJn0/wRS2SEyIqQZIgIJEdIMkQEkiOkPC4Dgu2T+UtDphwOdL6Nzii3ABJF1OujOy3POe4zkb4Qn7lsiTUyQsJESLJERAkbIckSETemUBHSMqhH5KdVDkY7Mkt74GfmY7mTRJAIGUFAIUUhIiSokKIQERlTICHpwmiafCDWaRrF75QtShFBwClblCKCgFM2LSJNvpAYGYEPIUUtIsSPkKIWERmTZyFpEWnyiUTJCDwISZWIEC9CUiUiMiZXIWkRafKNxMkIHISkWkSIk5BUi4iMyVZIWkSafCQxCWwRmNQ+fPhwqr6+Hrq7u5WLiEKT2g899BBcv34dXn/9deUi4saUS2rfVveN0rKazTDz8S+1iDR5SaJlBEsHWzsA9ABAPQD0m6bZH/N4Kq3xsNqRI9aYBmMeE/tuDgAA+65OsfumaV6Jc0wajV8SLyONRrM8SGTOSKPRLD+0jDQaTSLQMtJoNIlAy0ij0SQCLSONRpMItIw0Gk38AMD/A//VoOoJFDCLAAAAAElFTkSuQmCC"/>
```

Comment:
Hmm.

https://springload.slack.com/archives/C3JHNA6T0/p1668579172998559

This should be either a proper SVG or just use a png.

Reply:
changed